[CI] GraphExploreResponseTests#testHlrcFromXContent fails

https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.x+intake/353/console

Error:
```
FAILURE 0.81s | GraphExploreResponseTests.testHlrcFromXContent <<< FAILURES!
   > Throwable #1: java.lang.AssertionError: Error when comparing xContent
[...]
at __randomizedtesting.SeedInfo.seed([2EB6774B7153F34E:9216D58F28F4F8C2]:0)
   >    at org.elasticsearch.test.hamcrest.ElasticsearchAssertions.assertToXContentEquivalent(ElasticsearchAssertions.java:678)
   >    at org.elasticsearch.test.AbstractXContentTestCase$XContentTester.test(AbstractXContentTestCase.java:125)
   >    at org.elasticsearch.test.AbstractXContentTestCase.testFromXContent(AbstractXContentTestCase.java:182)
```

This also reproduces locally for me on master
```
./gradlew :x-pack:plugin:core:test \
  -Dtests.seed=2EB6774B7153F34E \
  -Dtests.class=org.elasticsearch.protocol.xpack.graph.GraphExploreResponseTests \
  -Dtests.method="testHlrcFromXContent" \
  -Dtests.security.manager=true \
  -Dtests.locale=de \
  -Dtests.timezone=Europe/Vaduz \
  -Dcompiler.java=11 \
  -Druntime.java=8
```

This one failed again in https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+master+periodic/576/console and reproduces locally also 

```
20:42:10 FAILURE 0.05s J5 | GraphExploreResponseTests.testHlrcFromXContent <<< FAILURES!
20:42:10    > Throwable #1: java.lang.AssertionError: Error when comparing xContent.
20:42:10    >                    connections: 
```
```
20:42:10    > 	at __randomizedtesting.SeedInfo.seed([464BC7502D9463F5:FAEB659474336879]:0)
20:42:10    > 	at org.elasticsearch.test.hamcrest.ElasticsearchAssertions.assertToXContentEquivalent(ElasticsearchAssertions.java:693)
20:42:10    > 	at org.elasticsearch.test.AbstractXContentTestCase$XContentTester.test(AbstractXContentTestCase.java:139)
20:42:10    > 	at org.elasticsearch.test.AbstractXContentTestCase.testFromXContent(AbstractXContentTestCase.java:196)
20:42:10    > 	at org.elasticsearch.protocol.AbstractHlrcXContentTestCase.testHlrcFromXContent(AbstractHlrcXContentTestCase.java:21)
20:42:10    > 	at java.lang.Thread.run(Thread.java:748)
20:42:10    > Caused by: java.lang.AssertionError: expected:<33> but was:<84>
```

REPRO with 
```
./gradlew :x-pack:plugin:core:unitTest -Dtests.seed=464BC7502D9463F5 -Dtests.class=org.elasticsearch.protocol.xpack.graph.GraphExploreResponseTests -Dtests.method="testHlrcFromXContent" -Dtests.security.manager=true -Dtests.locale=es-EC -Dtests.timezone=Etc/GMT-8 -Dcompiler.java=11 -Druntime.java=8
```

--- a/x-pack/plugin/core/src/test/java/org/elasticsearch/protocol/AbstractHlrcXContentTestCase.java
+++ b/x-pack/plugin/core/src/test/java/org/elasticsearch/protocol/AbstractHlrcXContentTestCase.java
@@ -21,7 +21,7 @@ public abstract class AbstractHlrcXContentTestCase<T extends ToXContent, H> exte
         AbstractXContentTestCase.testFromXContent(NUMBER_OF_TEST_RUNS, this::createTestInstance, supportsUnknownFields(),
             getShuffleFieldsExceptions(), getRandomFieldsExcludeFilter(), this::createParser,
             p -> convertHlrcToInternal(doHlrcParseInstance(p)),
-            this::assertEqualInstances, true, getToXContentParams());
+            this::assertEqualInstances, assertToXContentEquivalence(), getToXContentParams());
     }
 
     /**

